### PR TITLE
Accessors for image properties in manipulation

### PR DESCRIPTION
**Pull request by anawu2006**
https://app.asana.com/0/1130320099638868/1134361291187905/f

- Add accessor for original image scale amount.
- Add setters for isAnimatedZooming and isInFling.

### CODE REVIEW

**Comment by jgavris** [comment]
I'm ok with merging Larry's commit in here too to get back on track for using pull requests against this sub project.

**Comment by jgavris** [comment]
@larryng can you review?

File: photoview/src/main/java/com/github/chrisbanes/photoview/PhotoViewAttacher.java [hunk 1]
```diff
@@ -314,6 +321,14 @@ public float getScale() {
             (getValue(mSuppMatrix, Matrix.MSKEW_Y), 2));
     }
 
+    /**
+     * Amount original image is scaled, where 1 is no scaling.
+     */
+    public float getImageScale() {
+        return (float) Math.sqrt((float) Math.pow(getValue(mDrawMatrix, Matrix.MSCALE_X), 2) + (float) Math.pow
```
Comment (larryng) [inline photoview/src/main/java/com/github/chrisbanes/photoview/PhotoViewAttacher.java]:
```java
        return (float) Math.sqrt(
                Math.pow(getValue(mDrawMatrix, Matrix.MSCALE_X), 2) +
                        Math.pow(getValue(mDrawMatrix, Matrix.MSCALE_Y), 2)
        );
```

Comment (larryng) [reply photoview/src/main/java/com/github/chrisbanes/photoview/PhotoViewAttacher.java]: Uh, never mind.

Comment (larryng) [reply photoview/src/main/java/com/github/chrisbanes/photoview/PhotoViewAttacher.java]: I think `getScale()` and `getImageScale()` can be cleaned up (unnecessary type coercion, readability), but that's the library's code so whatever.

Comment (anawu2006) [reply photoview/src/main/java/com/github/chrisbanes/photoview/PhotoViewAttacher.java]: MSKEW_Y takes rotation into consideration as well.

**Pull request merged**